src/resources/postcss/tribe-common-admin.pcss

```diff
@@ -129,11 +129,17 @@ table.plugins .tribe-plugin-update-message a {
 		margin-top: 0;
 	}
 }
+.tribe-settings-form .tribe-field-checkbox_bool p.tooltip {
```

Comment:
Around this element there a couple of uses of `.tribe-settings-form` to then include a child rule. It would be amazing to move the child rules into a:

```
.tribe-settings-form {
    // .child-rules....
}
```

Reply:
Nested quite a few I found. Good suggestion.